test/fluree/db/query/filter_query_test.clj

```diff
@@ -0,0 +1,69 @@
+(ns fluree.db.query.filter-query-test
```

Comment:
Do we have some established convention for how we're splitting up/organizing our test ns's here? I put this in its own test file (and made it clj-only), since that seems to be the pattern (we've already got eg `aggregate-test`, `optional-query-test`), but it was just a guess. 

I'm a little puzzled by, for example, `misc-queries-test`, `json-ld-basic-test`, and `json-ld-compound-test`.

Reply:
No established convention.